src/import/engine.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::i32;
+use std::fmt;
+use std::ops::Deref;
+use std::sync::Arc;
+use std::path::Path;
+
+use uuid::Uuid;
+
+use rocksdb::{BlockBasedOptions, ColumnFamilyOptions, DBOptions, Writable, WriteBatch as RawBatch,
+              DB};
+use kvproto::importpb::*;
+
+use config::DbConfig;
+use storage::CF_DEFAULT;
+use storage::types::Key;
+use util::config::MB;
+use util::rocksdb::{new_engine_opt, CFOptions};
+use util::rocksdb::properties::SizePropertiesCollectorFactory;
+
+use super::Result;
+
+pub struct Engine {
+    db: Arc<DB>,
+    uuid: Uuid,
+}
+
+impl Engine {
+    pub fn new<P: AsRef<Path>>(path: P, uuid: Uuid, opts: DbConfig) -> Result<Engine> {
+        let db = {
+            let (db_opts, cf_opts) = tune_dboptions_for_bulk_load(&opts);
```

Comment:
do you only import to the default CF?

Reply:
No, the data for the write CF will be generated from the default CF when we actually import the SST files.

Reply:
so here no need to change the Write CF options?

Reply:
Yes